images/public-log-vendor-matcher/pg-init.d/lib-16-stop-postgres.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# Stop Postgres
+
+[ ! "${ASN_DATA_PIPELINE_RETAIN:-}" = true ] && \
+# in the Postgres container image,
+# the command run changes to "postgres" once it's completed loading up
+# and is in a ready state and all of the init scripts have run
+#
+# here we wait for that state and attempt to exit cleanly, without error
```

Comment:
... and we do that because ___ ?  The answer to this also probably belongs in the README

Reply:
Updated https://github.com/kubernetes/k8s.io/pull/2710/commits/c50490d7defdf48e79f1020326d3884fce9a9210

Reply:
I think I'm looking for answer like "postgres now stays up and running, but we're running this container as a one-shot job and using pg-init.d as the driver for all of the logic in this container. Once the command changes to postgres, we know pg-init.d has finished executing all of our scripts and sql, so we're ready to exit this container cleanly to signify to prow or whatever CI system runs this that that the import/transform has finished" but cleaned up to be less rambling